Assets/MixedRealityToolkit/Providers/Hands/HandRay/HandRay.cs

```diff
@@ -110,80 +111,4 @@ private Vector3 ComputeRayPivotPosition(Vector3 handPosition, Transform headTran
             return headTransform.position + headRotationFlat * relativePivot;
         }
     }
-
-    /// <summary>
-    /// A ray whose position and direction are stabilized in a way similar to how gaze stabilization
-    /// works in HoloLens.
-    /// 
-    /// The ray uses Anatolie Gavrulic's "DynamicExpDecay" filter to stabilize the ray
-    /// this filter adjusts its smoothing factor based on the velocity of the filtered object
-    /// 
-    /// The formula is
-    ///  Y_smooted += ∆𝑌_𝑟
-    ///  where
-    /// 〖∆𝑌_𝑟=∆𝑌∗〖0.5〗^(∆𝑌/〖Halflife〗)
-    /// 
-    /// In code, LERP(signal, oldValue, POW(0.5, ABS(signal – oldValue) / hl)
-    /// </summary>
-    internal class StabilizedRay
-    {
-        public float HalfLifePosition { get; } = 0.1f;
-
-        public float HalfLifeDirection { get; } = 0.1f;
-
-        public Vector3 StabilizedPosition { get; private set; }
-
-        public Vector3 StabilizedDirection { get; private set; }
-
-        public Quaternion StabilizedRotation => Mathf.Abs(StabilizedDirection.magnitude) < 1e-5f ?
```

Comment:
Was this left out of the utilities class intentionally?

Reply:
It wasn't used and is a shorthand with odd results. 
It computes a magnitude check, and abs, and returns a look rotation, which can easily be done outside. If I were to re-add it, I'd cache the rotation at least, instead of computing it on each access.